com.ibm.wala.cast.js.rhino/build.gradle

```diff
@@ -8,6 +6,42 @@ dependencies {
 			project(':com.ibm.wala.core'),
 			project(':com.ibm.wala.util'),
 	)
+	testImplementation(
+			'junit:junit:4.13',
+			testFixtures(project(':com.ibm.wala.cast')),
+			testFixtures(project(':com.ibm.wala.cast.js')),
+	)
+	testFixturesImplementation(
+//			'commons-io:commons-io:2.6',
```

Comment:
Maybe just remove commented-out deps?

Reply:
Good catch. Commenting out was for quick experiments; I always intended to remove the unneeded dependencies entirely once I was sure I could. Now removed in 0817d538337e6e1c5a47713938fdcb0f1b264db8.